src/lib/net/index.ts

```diff
@@ -0,0 +1,60 @@
+import { RouterClass } from "./classes/router";
+import { RepositoryClass } from "./classes/directory";
+
+/* Types */
+import {
+	BaseEndpoints,
+	RepositoryDeclaration,
+	RepositoryObjectDeclaration,
+	EndpointsDeclaration,
+	RouterDeclaration,
+} from "./classes/types";
+
+/* POST */
+import { PostEndpoint } from "./classes/methods/post";
+import { POSTDeclaration } from "./classes/methods/postTypes";
+
+/* UPDATE */
+
+/* GET */
+
+export namespace Network {
+	/**
+	 * Network.Method is a namespace containing all the available methods that Tina offers to work with the networking layer.
+	 */
+	export namespace Method {
+		/**
+		 * POST is a network method which lets you send data to the server, this doesn't expect anything back. Shouldn't be used careless.
+		 *
+		 * @param identifier possible unique identifier, can be empty.
+		 * @returns a POST Endpoint.
+		 */
+		export function POST<T extends unknown[] = unknown[]>(identifier?: string): POSTDeclaration<T> {
+			return new PostEndpoint<T>();
+		}
```

Comment:
Uh, did you forget the other methods?

Reply:
Noup, just wanted the single review on this one.